### ISSUE TITLE
Change "We define" to be "The Standard for Public Code defines" for the definition of public code?

### ISSUE THREAD

**Issue by Ainali** [publiccodenet/publiccode.net#290]
Should this say "We define" or change to be "The Standard for Public Code defines" ?

_Originally posted by @ericherman in https://github.com/publiccodenet/publiccode.net/issues/286#issuecomment-1697651340_

**Comment by ElenaFdR** [comment]
I don't think so. This is an introduction to both the concept of public code and the Foundation for someone who is unfamiliar with both. Adding a reference to the Standard sounds like we're trying to boost our legitimacy by referencing an external document...that we wrote. This isn't the right place for that complexity.
